### DISCUSSION TITLE
Question on exportWindow()

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#544] [Q&A] [answered]
Hi! 
I have been using `exportWindow()` and `importWindow()` to save and load the plots that I am making. I have also been adding callback functionality to the plots in buttons. I have noticed that when I load the saved plots with `importWindow()`, that I only get the data back and not the callback and button logic. I was just wondering if there is a way to save this as well, or if that has to be rebuilt in the load function. I am saving to `.npz` by the way...

Cheers 
j

**Comment by marcomusy** [comment, marked as answer]
Hi, ..no that is definitely not possible, because you have no way to save the actual code of the callback (well... not that I know of..).

**Reply by jkissi** [reply]
Ok, cheers!
j
